g2p_registry_addl_info/models/json_field.py

```diff
@@ -0,0 +1,17 @@
+import json
```

Comment:
Can we move json_field outside or registry. So that it can be in use independent of registry

Reply:
@shibu-narayanan ok sure. What's the best place to put this ?